.cspell.json

```diff
@@ -0,0 +1,39 @@
+{
+  "version": "0.2",
+  "gitignoreRoot": ".",
+  "dictionaries": [
+    "css",
```

Comment:
can you add `filetypes` and `npm` and `misc`, that should limit number of words in custom dictionary by alot

Reply:
Tried adding those locally, but there any additional words it removed. I think it was mostly covered by excluding the DOCX and XYZ file types